pages/accountLists/[accountListId]/contacts.page.tsx

```diff
@@ -1,29 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Head from 'next/head';
 import { useTranslation } from 'react-i18next';
 import { useRouter } from 'next/router';
 import { Box } from '@material-ui/core';
 import { ContactFilters } from '../../../src/components/Contacts/ContactFilters/ContactFilters';
 import { ContactsTable } from '../../../src/components/Contacts/ContactsTable/ContactsTable';
+import { ContactDetails } from '../../../src/components/Contacts/ContactDetails/ContactDetails';
 
 const ContactsPage: React.FC = () => {
   const { t } = useTranslation();
   const {
     query: { accountListId },
   } = useRouter();
 
+  const [contactDetailsId, setContactDetailsId] = useState<string>();
+
   return (
     <>
       <Head>
         <title>MPDX | {t('Contacts')}</title>
       </Head>
-      <Box height="100vh" display="flex">
-        <Box flex={1}>
+      <Box height="100vh" display="flex" overflow-y="scroll">
+        <Box width="20vw">
           <ContactFilters accountListId={accountListId as string} />
         </Box>
-        <Box flex={4}>
-          <ContactsTable accountListId={accountListId as string} />
+        <Box flex={1}>
+          <ContactsTable
+            accountListId={accountListId as string}
+            onContactSelected={setContactDetailsId}
+          />
         </Box>
+        {contactDetailsId ? (
+          <Box flex={1} hidden={!contactDetailsId}>
```

Comment:
Do you need the ```hidden``` if it doesn't get rendered unless ```contactDetailsId``` is true?

Reply:
Oh, I forgot to remove that from the previous way I implemented this.